aspnetcore/performance/response-compression/sample/Program.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace ResponseCompressionSample
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var host = new WebHostBuilder()

```

Comment:
Most of our samples have it in case you want to launch the sample behind with IIS or IISExpress even though it's not necessary in most cases to illustrate the usage. You are right in that  it shouldn't affect how the app runs it may be useful to also illustrate the web.config changes needed to disable dynamic compression. You can refer the web.config instead of writing up the code samples explicitly in the docs.